### PR TITLE
feat(news gallery): add release info

### PR DESCRIPTION
**Pull request by manojava-gk**
add new release info to the news gallery.
create new release page in blog section

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by danielmiehle — APPROVED** [review]

File: static/img/Tractus-X_Release24.03.png
Comment (maximilianong) [inline static/img/Tractus-X_Release24.03.png:1]: Thanks for updating!

**Review by maximilianong — APPROVED** [review]

File: utils/newsTitles.js [hunk 1]
```diff
@@ -8,40 +13,5 @@ export const newsTitles = [
     date: "19.02.2024",
     title: "Second Eclipse Tractus-X Community Days",
     blogLink: "/blog/community-days-05-2024"
-  },
```
Comment (stephanbcbauer) [inline utils/newsTitles.js:11]: Are these entries deleted on purpose? If so, why? Personal opinion -> I think it's nice to see what has happened and been communicated.

File: blog/2024-03-08-release-24-03.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+We are pleased to announce our first minor release in 2024!
+
+Tractus-X Release 24.03 focuses on improving the performance and stability of our Catena-X core and onboard services, strengthening our data space foundation.
+
+Use Cases & Value Creation: With five new KITs we are introducing exciting new offerings to solve the toughest industry problems such as company certificate management, data governance, manufacturing as a service, environmental & social standards (ESS).
+
+The Industry Core is an integrative layer between the industry-agnostic dataspace foundation and the use cases. It ensures that the various applications can access common functionalities. It describes how digital twins of components, materials and vehicles can be created for the automotive industry and linked with direct business partners. 
+
+Data Space Governance: The Catena-X Automotive Network e.V. will provide the corresponding standards (https://catena-x.net/de/standard-library).
```
Comment (stephanbcbauer) [inline blog/2024-03-08-release-24-03.md:25]: Isn't the Markdown linter complaining here? I would have expected the URL to look like this <https://catena-x.net/de/standard-library>

File: utils/newsTitles.js (hunk 1 above)
Comment (stephanbcbauer) [reply utils/newsTitles.js:11]: Forget my comment :) these are just the announcements on the homepage :) Thank you

Comment (manojava-gk) [reply utils/newsTitles.js:11]: @maximilianong suggested to keep only latest 3

Comment (maximilianong) [reply utils/newsTitles.js:11]: There were pretty old news entries :D

**Pull request merged**